internal/updater/updater_test.go

```diff
@@ -0,0 +1,457 @@
+package updater
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+)
+
+var (
+	schemeBuilder = runtime.NewSchemeBuilder(
+		kscheme.AddToScheme,
+		rukpakv1alpha1.AddToScheme,
+	)
```

Comment:
nit: can we avoid polluting global state and define this in the NewFakeUpdater call site?

Reply:
Yes, we can.  I'll move it to the caller place.  Thanks!